src/UmaCtfAdapter.sol

```diff
@@ -139,8 +140,7 @@ contract UmaCtfAdapter is IUmaCtfAdapter, Auth, BulletinBoard, IOptimisticReques
         bytes32 questionID = keccak256(ancillaryData);
         QuestionData storage questionData = questions[questionID];
 
-        // Upon dispute, immediately reset the question, sending out a new price request
-        // paying for the price request from the Adapter's balance
+        // Upon dispute, immediately reset the question, which creates a new OO price request paid by the Adapter
         _reset(address(this), questionID, questionData);
```

Comment:
Can check here that `reset == true`. Would be a little bit easier to read off the logic here.